modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_request.rb

```diff
@@ -8,7 +8,8 @@ class PowerOfAttorneyRequest < ApplicationRecord
 
     has_one :power_of_attorney_form,
             inverse_of: :power_of_attorney_request,
-            required: true # for now
+            required: true, # for now
+            dependent: :destroy
```

Comment:
Does this cause ActiveRecord to destroy POA forms when POA requests are destroyed? If so,
- We won't be deleting POA requests
- Relying on models' `destroy` which runs all of AR's lifecycle callback code and such is a pretty heavy approach